packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -35,7 +35,7 @@ export interface GraphNGProps extends Omit<PlotProps, 'data' | 'config'> {
 
 const defaultConfig: GraphFieldConfig = {
   drawStyle: DrawStyle.Line,
-  points: PointMode.Auto,
+  showPoints: ShowPoints.Auto,
```

Comment:
points might be bad, but feel `showPoints` is not much better, as it signifies a verb/action, not an option. 

* linePoints  (LinePointsMode for enum) 
* linePointMode (LinePointsMode for enum)

Reply:
the problem with a "line" prefix is that there's not a line rendered. points can also be shown on bars (or lizards). i was thinking about using a single "pointDensity" which would map to uPlot's `points.space`. this gives the most control, where:

```js
null | "" -> "Auto"
0 -> "Always"
-1 -> "Never"
100 -> 100px
```

the UI can still hide the density aspect unless we ever want to expose it.

Reply:
i the current UI setup, "when/if points are shown" is exactly what this setting semantically does. "mode" is ambiguous, and "line" is orthogonal. i see you point, though. the naming makes more sense when viewed only through the JSON output, and is not as straightforward if you have to use it in the code.

Reply:
"pointVisibility" - a bit wordy, but might work better.

Reply:
the old graph uses `showPoints`, which i think is pretty clear. 

![image](https://user-images.githubusercontent.com/43234/101258595-1f93ec80-36e9-11eb-85fa-4b384f4fb79e.png)

would you be okay with changing the enum name to `PointVisibility` but leaving the option as `showPoints`?

Reply:
Yea, it works ok. Reason I suggested linePoints is that it is only enabled/relevant when line drawStyle is set

Reply:
ok, cool. i think it's more flexible to leave it unprefixed and omit "line". especially since we have a dedicated "Points" mode where all the line options are disabled.

Reply:
But is not an argument for the line prefix as it's only relevant for line? But I guess it be relevant in other future non point drawStyle options :)

Reply:
i see what you did there; touché, good sir!